Formula/sync_gateway.rb

```diff
@@ -16,10 +15,41 @@ class SyncGateway < Formula
   end
 
   depends_on "go" => :build
+  depends_on :gpg => :build
+
+  resource "depot_tools" do
+    url "https://chromium.googlesource.com/chromium/tools/depot_tools.git",
+      :revision => "935b93fb9bf367510eece7db8ee3e383b101c36d"
+  end
 
   def install
-    system "make", "buildit"
-    bin.install "bin/sync_gateway"
+    ENV["GOPATH"] = buildpath/"godeps"
+    repo_cache = HOMEBREW_CACHE/"repo_cache/#{name}/.repo"
+    repo_cache.mkpath
```

Comment:
Yeah, sync_gateway, like v8, uses https://chromium.googlesource.com/chromium/tools/depot_tools.git. So this is to support caching with that vendor system without having to resort to `go_resource` blocks.